test/quantization/test_quantize_fx.py

```diff
@@ -574,45 +577,91 @@ def forward(self, x):
         original_ref_m.conv2.weight = torch.nn.Parameter(original_m.standalone.conv.weight.detach())
         original_ref_m.conv2.bias = torch.nn.Parameter(original_m.standalone.conv.bias.detach())
 
-        qconfig_dict = {"": default_qconfig}
-        config_name = {"standalone_module_name": [("standalone", None, None)]}
-        config_class = {"standalone_module_class": [(StandaloneModule, None, None)]}
-        for prepare_config in [config_name, config_class]:
+        float_interface_config = {
+            "input_quantized_idxs": [], # float input
+            "output_quantized_idxs": [], # float output
+        }
+
+        quantized_interface_config = {
+            "input_quantized_idxs": [0], # quantized input
+            "output_quantized_idxs": [0], # quantized output
+        }
+        # is_name, is_float
+        options = itertools.product([True, False], [True, False])
+        for is_name, is_float in options:
+            if is_float:
+                interface_config = float_interface_config
+                # input and output of first conv, observer for standalone module
+                # will be inserted in the standalone module itself
+                prepare_count_check = {
+                    ns.call_module(torch.quantization.MinMaxObserver): 2
+                }
+                # for input and output of conv in the standalone module
+                standalone_prepare_count_check = {
+                    ns.call_module(torch.quantization.MinMaxObserver): 2
+                }
+                convert_count_check = {
+                    ns.call_function(torch.quantize_per_tensor) : 1,
+                    ns.call_module(nnq.Conv2d) : 1,
+                    ns.call_method("dequantize") : 1,
+                }
+                standalone_convert_count_check = {
+                    # standalone module will take float as input and output
+                    # so we'll see quantize and dequantize in the modoule
+                    ns.call_function(torch.quantize_per_tensor) : 1,
+                    ns.call_module(nnq.Conv2d): 1,
+                    ns.call_method("dequantize") : 1,
+                }
+            else:
```

Comment:
When inputs and outputs are quantized, why do we expect to see quantize_per_tensor and dequantize?

Reply:
which one are you referring to? in standalone_convert_count_check we didn't expect to see quantize_per_tensor or dequantize. we only expect to see them in parent (convert_count_check)

Reply:
Got it, thanks